pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -74,6 +74,8 @@ var (
 )
 
 type vmiCountMetric struct {
```

Comment:
shouldn't this change to`vmiInfoMetric`?

Reply:
Actually, I think it can be removed.

Reply:
I deleted the usage, because it was no longer aggregated here, need to also cleanup the type yes

Reply:
updated